spec/views/hq/orphans/show_spec.rb

```diff
@@ -11,7 +11,9 @@
       expect(rendered).to match @orphan.name
```

Comment:
We have that syntax elsewhere in the suite but it still gives failures on apostrophes